server/routes/makeAReferral/makeAReferralController.ts

```diff
@@ -153,11 +153,31 @@ export default class MakeAReferralController {
     res.redirect(`/referrals/${req.params.id}/risk-information`)
   }
 
+  private async getDraftOasysRiskInformation(
```

Comment:
does this function exist anywhere else? i'm sure i've seen it / reviewed it before?

Reply:
It exists in interventionsService.getDraftOasysRiskInformation

The purpose of this method is to convert NOT_FOUND into null rather than throw an exception.

An alternative would be to modify `interventionsService.getDraftOasysRiskInformation` to return null in case of NOT_FOUND.